client/src/components/timeline/Calendar.vue

```diff
@@ -135,6 +136,12 @@ export default {
       const newColumnWidth = this.columnWidth + level * zoomFactor;
       this.setPixelsPerDay(newColumnWidth);
       this.$el.scrollTo(daysOutsideViewport * newColumnWidth, 0);
+    },
+    jumptotoday() {
+    const today = moment();
+    const indexOfToday = this.days.findIndex(day => day.isSame(today, "d"));
+    const calendarWidth = this.$el.clientWidth;
+    this.$el.scrollTo(indexOfToday * this.columnWidth - calendarWidth / 3, 0);
```

Comment:
Duplicate code: You could use the method ‘scrollToDate’, which is defined a few lines above. It would also be good, if the same method would be used on mount (‘this.scrollToToday()’ or similar).

Reply:
Doesn't work (since #8 was merged)